kpatch-build/create-diff-object.c

```diff
@@ -654,6 +680,21 @@ static int kpatch_line_macro_change_only(struct section *sec)
 }
 #endif
 
+static bool kpatch_changed_child_needs_parent_profiling(struct symbol *sym)
```

Comment:
I'll add the comment.

Superfunction sounds a bit odd. But subfunction definitely sounds better than child.
Maybe Top function?
Otherwise I actually like the "part" terminology (without being a direct references to the .part. in the name of the symbol). So I would suggest e.g. "function_parts" and "part_of" .

Reply:
I'm not sure about function_parts/part_of.  Maybe parent/child is ok.